localization/english/culture/wc_cultures_l_english.yml

```
@@ -0,0 +1,402 @@
+ stormreaver:0 "Stormreaver"
+ stormreaver_collective_noun:1 "Stormreavers"
+ thunderlord:0 "Thunderlord"
+ thunderlord_collective_noun:1 "Thunderlords"
+ bonechewer:0 "Bonechewer"
+ bonechewer_collective_noun:1 "Bonechewers"
+ laughting_skull:0 "Laughing Skull"
+ laughting_skull_collective_noun:1 "Laughing Skulls"
+ shattered_hand:0 "Shattered Hand"
+ shattered_hand_collective_noun:1 "Shattered Hands"
+ frostwolf:0 "Frostwolf"
+ frostwolf_collective_noun:1 "Frostwolves"
+ twilights_hammer:0 "Twilight's Hammer"
+ twilights_hammer_collective_noun:1 "Twilights"
+ dragonmaw:0 "Dragonmaw"
+ dragonmaw_collective_noun:1 "Dragonmaws"
+ durotaric:0 "Durotarian"
+ durotaric_collective_noun:1 "Durotarians"
+ maghar:0 "Mag'har"
+ maghar_collective_noun:1 "Mag'har"
+ moknathal:0 "Mok'nathal"
+ moknathal_collective_noun:1 "Mok'nathal"
+
+ #Titanforged group
+ titanforged_group:0 "Titanforged"
+ titanforged_group_collective_noun:1 "Titanforged"
+ titanforged:0 "Titanforged"
+ titanforged_collective_noun:1 "Titanforged"
+ frozen_giant:0 "Frost Giant"
+ frozen_giant_collective_noun:1 "Frost Giants"
+ tolvir:0 "Tol'vir"
+ tolvir_collective_noun:1 "Tol'vir"
+ mechagnome:0 "Mechagnome"
+ mechagnome_collective_noun:1 "Mechagnomes"
+ earthen:0 "Earthen"
+ earthen_collective_noun:1 "Earthen"
+ iron_dwarf:0 "Iron Dwarf"
+ iron_dwarf_collective_noun:1 "Iron Dwarves"
+ titan:0 "Titan"
+ titan_collective_noun:1 "Titans"
+ storm_giant:0 "Storm Giant"
+ storm_giant_collective_noun:1 "Storm Giants"
+
+ #Demonic group
+ demonic_group:0 "Demonic"
+ demonic_group_collective_noun:1 "Demons"
+ eredar:0 "Eredar"
+ eredar_collective_noun:1 "Eredar"
+ annihilan:0 "Annihilan"
+ annihilan_collective_noun:1 "Annihilans"
+ nathrezim:0 "Nathrezim"
+ nathrezim_collective_noun:1 "Nathrezim"
+ eredruin:0 "Ered'ruin"
+ eredruin_collective_noun:1 "Ered'ruin"
+ shivarra:0 "Annihilan"
+ shivarra_collective_noun:1 "Annihilans"
+ moarg:0 "Nathrezim"
+ moarg_collective_noun:1 "Nathrezim"
+ observer:0 "Ered'ruin"
+ observer_collective_noun:1 "Ered'ruin"
+ sayaadi:0 "Sayaadi"
+ sayaadi_collective_noun:1 "Sayaadi"
+
+ #Dwarven group
+ dwarf_group:0 "Dwarven"
+ dwarf_group_collective_noun:1 "Dwarves"
+ bronzebeard:0 "Bronzebeard"
+ bronzebeard_collective_noun:1 "Bronzebeards"
+ wildhammer:0 "Wildhammer"
+ wildhammer_collective_noun:1 "Wildhammers"
+ dark_iron:0 "Dark Iron"
+ dark_iron_collective_noun:1 "Dark Irons"
+ old_dwarven:0 "Dwarven"
+ old_dwarven_collective_noun:1 "Dwarves"
+ frostborn:0 "Frostborn"
+ frostborn_collective_noun:1 "Frostborn"
+
+ #Elven group
+ elf_group:0 "Elven"
+ elf_group_collective_noun:1 "Elves"
+ high_elf:0 "High Elf"
+ high_elf_collective_noun:1 "High Elves"
+ blood_elf:0 "Blood Elf"
+ blood_elf_collective_noun:1 "Blood Elves"
+ night_elf:0 "Night Elf"
+ night_elf_collective_noun:1 "Night Elves"
+ highborne:0 "Highborne"
+ highborne_collective_noun:1 "Highborne"
+ nightborne:0 "Nightborne"
+ nightborne_collective_noun:1 "Nightborne"
+ naga:0 "Naga"
```

Back on CK2 it was...
```suggestion
 naga:0 "Nazja"
```